.vscode/launch.json

```diff
@@ -5,6 +5,17 @@
 {
   "version": "0.2.0",
   "configurations": [
+    {
```

Comment:
Looks like this was copied from the replayer sample.
Let's keep it for now but it'll probably need to be modified when we get to launching the replayer.